packages/breadcrumbs/Breadcrumbs.js

```diff
@@ -49,16 +49,40 @@ Breadcrumbs.propTypes = {
   active: PropTypes.string.isRequired,
   emptyState: PropTypes.string,
   children: (props, propName, componentName) => {
-    const prop = props[propName];
-
+    if (!props.children) {
+      return null;
+    }
     let error = null;
-    React.Children.forEach(prop, child => {
-      if (child.type !== BreadcrumbItem) {
+
+    // multi children
+    if (Array.isArray(props.children)) {
```

Comment:
Can we add a corresponding test case using `react-testing-library`?

Reply:
DONE!